pybamm/parameters/lithium_ion_parameters.py

```diff
@@ -535,6 +535,12 @@ def _set_parameters(self):
         eps_c_init_av = pybamm.xyz_average(
             self.epsilon_s * pybamm.r_average(self.c_init)
         )
+        # if self.options['open-circuit potential'] == 'Plett':
+        self.K = pybamm.Parameter(f"{pref}{Domain} particle hysteresis decay rate")
+        self.K_x = pybamm.Parameter(
+            f"{pref}{Domain} particle hysteresis switching factor"
+        )
```

Comment:
this name should be more informative as well since `K` is not a standard abbreviation in battery modeling

Reply:
Done